packages/api/tests/endpoints/test-granules.js

```diff
@@ -1659,9 +1659,9 @@ test.serial('update (PUT) returns bad request if granule does not exist', async
     .set('Authorization', `Bearer ${jwtAuthToken}`)
     .set('Accept', 'application/json')
     .send(newGranule)
-    .expect(400);
+    .expect(404);
```

Comment:
Please update the test title `bad request` to `not found`.

Reply:
dadaa7afed